data/brands/24s.yaml

```diff
@@ -0,0 +1,22 @@
+name: 24S
+status: avoid
+description: "**24S** is owned by **LVMH**.\n\nBernard Arnault, the chairman and **CEO**\
+  \ of **LMVH** Moet Hennessy-Louis Vuitton, and the world\u2019s richest man according\
+  \ to Forbes, has **invested** in cybersecurity firm, **Wiz**, the **Israeli** maker\
+  \ of cloud security solutions for enterprises."
+reasons:
+- TBD
```

Comment:
given the high number of TBDs, maybe it would be better to leave them blank or empty.. like the one for drinkmate:
https://github.com/TechForPalestine/boycott-israeli-consumer-goods-dataset/blob/main/data/brands/drinkmate.yaml

wdyt?

Reply:
CI is failing with:
```
Python 3.12.1
ERROR:root:Validation error in ralphlaurenfragrances
ERROR:root:None is not of type 'string'

Failed validating 'type' in schema['properties']['logo_url']:
Validating 525 brands
    {'description': 'Logo should be at least 200x200 pixels',
     'format': 'uri',
     'pattern': '^https?://',
     'type': 'string'}

On instance['logo_url']:
    None
```

such errors should go away if you omit empty/None - making the decision for you 😁

Reply:
Thanks for fixing this with your PR :)